controls/togglebutton/getting-started.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Getting Started
+page_title: .NET MAUI ToggleButton Documentation - Getting Started
+description: Get started with the Telerik UI for .NET MAUI ToggleButton control and add the control to your .NET MAUI project.
+position: 2
+slug: togglebutton-getting-started
+---
+
+# Getting Started with the .NET MAUI ToggleButton
+
+This guide provides the information you need to start using the Telerik UI for .NET MAUI ToggleButton by adding the control to your project.
+
+At the end, you will achieve the following result.
+
+![.NET MAUI ToggleButton Getting Started](images/button-getting-started.png)
+
+## Prerequisites
+
+Before adding the ToggleButton, you need to:
+
+1. [Set up your .NET MAUI application]({%slug maui-getting-started %}#step-1-set-up-your-net-maui-application).
+
+1. [Download Telerik UI for .NET MAUI]({% slug maui-getting-started %}#step-2-download-telerik-ui-for-net-maui).
+
+1. [Install Telerik UI for .NET MAUI]({%slug maui-getting-started %}#step-3-install-telerik-ui-for-net-maui).
+
+## Define the Control
+
+**1.** When the your .NET MAUI application is set up, you are ready to add a ToggleButton control to your page.
+
+<snippet id='togglebutton-gettingstarted-xaml' />
+<snippet id='togglebutton-gettingstarted-csharp' />
+
+**2.** Add the `telerik` namespace:
+
+```XAML
+xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
+```
+```C#
+using Telerik.Maui.Controls;
+```
+
+**3.** Then define the Click event handler:
```

Comment:
I feel like an idiot but I can't figure out how is this tied to anything. In the sample for the Clicked event, it was included in the button definition. Just flagging in case some piece of code is missing.

```suggestion
**3.** Then define the `Clicked` event handler:
```